### DISCUSSION TITLE
how to display a field from a related table?

### DISCUSSION THREAD

**Discussion by juanzu** [rappasoft/laravel-livewire-tables#640] [Q&A]
I am trying to display a field from a related table but it only brings me the id. If I put company_id.name I get an error. a prompt response is appreciated

**in my component:**
```
  public function columns(): array
    {
        return [
            Column::make('Nombre', 'nombre')
            ->sortable()
            ->searchable(),
        Column::make('Acceso', 'acceso')
            ->sortable()
            ->searchable(),
            Column::make('Empresa', 'empresa_id')
            ->sortable()
            ->searchable(),
        Column::make('Correo', 'email')
            ->sortable()
            ->searchable(),
        Column::make('Estado', 'estado')
            ->sortable()
            ->searchable(),
        Column::make('Acciones')
            ->format(function ($value, $column, $row) {
                return view('livewire.usuarios.acciones')->withUser($row)->with('empresas');
            })
        ];
    }
```

**Comment by khedira00** [comment]
public function query() : Builder
    {
        $query = Product::with('restaurant');

        return $query;
    }

            Column::make(__('Restaurant Name'),'restaurant.name')
                ->searchable()
                ->sortable(function(Builder $query, $direction) {
                    return $query->select('products.*')->join('restaurants', 'products.restaurant_id', '=', 'restaurants.id')->groupBy('products.id')->orderBy('restaurants.name',$direction);
                }),

Somehing like that
